service/worker/workerdeployment/util.go

```diff
@@ -88,12 +88,14 @@ const (
 	errVersionAlreadyExistsType   = "errVersionAlreadyExists"
 	errMaxTaskQueuesInVersionType = "errMaxTaskQueuesInVersion"
 	errVersionAlreadyCurrentType  = "errVersionAlreadyCurrent"
-	errVersionIsDraining          = "Version cannot be deleted since it is draining."
 	errVersionNotFound            = "Version not found in deployment"
-	errVersionHasPollers          = "Version cannot be deleted since it has active pollers."
-	errVersionIsCurrentOrRamping  = "Version cannot be deleted since it is current or ramping."
-	errConflictTokenMismatchType  = "errConflictTokenMismatch"
-	errFailedPrecondition         = "FailedPrecondition"
+
+	errConflictTokenMismatchType = "errConflictTokenMismatch"
+	errFailedPrecondition        = "FailedPrecondition"
```

Comment:
nit: if some of these are public, it's probably worth just making them all public. no need to change it now though, just pointing out.